### PR TITLE
some typo fix

### PR DESCRIPTION
**Pull request by liwjGhostcloud**
there are some typo fix

### CODE REVIEW

File: pkg/apis/kops/cluster.go [hunk 1]
```diff
@@ -384,7 +384,7 @@ type ClusterSubnetSpec struct {
 	Egress string `json:"egress,omitempty"`
 	// Type define which one if the internal types (public, utility, private) the network is
 	Type SubnetType `json:"type,omitempty"`
-	// PublicIP to attatch to NatGateway
+	// PublicIP to attach to NatGateway
 	PublicIP string `json:"publicIP,omitempty"`
```
Comment (mikesplain) [inline pkg/apis/kops/cluster.go]: Great finds! Can you add these to the other versions of this file:
https://github.com/kubernetes/kops/blob/master/pkg/apis/kops/v1alpha1/cluster.go
https://github.com/kubernetes/kops/blob/master/pkg/apis/kops/v1alpha2/cluster.go

This helps keeping the docs for the different versions the same!

File: pkg/apis/kops/v1alpha1/instancegroup.go [hunk 2]
```diff
@@ -89,7 +89,7 @@ type InstanceGroupSpec struct {
 	Kubelet *KubeletConfigSpec `json:"kubelet,omitempty"`
 	// Taints indicates the kubernetes taints for nodes in this group
 	Taints []string `json:"taints,omitempty"`
-	// AdditionalUserData is any aditional user-data to be passed to the host
```
Comment (mikesplain) [inline pkg/apis/kops/v1alpha1/instancegroup.go]: Can you check to see if this mistake also exists on: 
https://github.com/kubernetes/kops/blob/master/pkg/apis/kops/v1alpha2/instancegroup.go
https://github.com/kubernetes/kops/blob/master/pkg/apis/kops/instancegroup.go

**Comment by chrislovecnm** [comment]
/lgtm cancel

@mikesplain I think you had some comments, and I removed the lgtm so they can be addressed.

**Comment by liwjGhostcloud** [comment]
when i exec `apiserver-boot build docs --disable-delegated-auth=false --output-dir docs/apireference --server kops-server` ,they were told me 
`cp: cannot create symbolic link '/build/node_modules/.bin/marked': Operation not supported
cp: cannot create symbolic link '/build/node_modules/.bin/mime': Operation not supported
cp: cannot create symbolic link '/build/node_modules/.bin/static': Operation not supported`
@mikesplain

**Comment by chrislovecnm** [comment]
@liwjGhostcloud  we broken the docs api generator when we moved to dep.  If you could help with that, it would be amazing!

**Comment by justinsb** [comment]
Going to merge and fix-up any api docs problems.  Thanks for the fixes @liwjGhostcloud 

/approve
/lgtm

**Pull request merged**